genesyscloud/resource_genesyscloud_webdeployments_deployment_test.go

```diff
@@ -0,0 +1,213 @@
+package genesyscloud
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
```

Comment:
This log import is unused and giving me a compile error because Go is fussy like that :)